lib/cassandra/utils.rb

```diff
@@ -1,3 +1,4 @@
 require_relative 'utils/cli/base'
 require_relative 'utils/daemon'
 require_relative 'utils/stats'
+require_relative 'utils/autoclean'
```

Comment:
I moved the `Autoclean` class into a "task" folder in 67059eaf95df85a600c7a5410898743d58fce813.